app/assets/stylesheets/components/_auth-form.scss

```diff
@@ -0,0 +1,80 @@
+.auth-form {
+  width: 100%;
+  max-width: 450px;
+  margin: auto 0;
+
+  &__wrapper {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 40px 50px 20px;
```

Comment:
Should we store this value in the `_variable` file?
Because it will be easier to manage all variables in one place.
Please apply this to the other line also 🙌

This is how to deal with variables:
```scss
// overrides bootstrap variables (if we use the bootstrap)
$paragraph-margin-bottom: 0;

// our custom variable
$_paragraph-line-height: 1.5;
```

Reply:
I've changed the line you've pointed at 😄 Updated in https://github.com/aetheryna/google-scraper-ruby/pull/32/commits/6e5051013e49fbd4fc6fe209042953850e62b12d. Couldn't find any other line like this though 😖